compiler/plugins/target/AMD-AIE/iree-amd-aie/Target/AMDAIETargetCDODirect.cpp

```diff
@@ -71,10 +71,8 @@ LogicalResult configureLocksAndBd(Block &block, const TileLoc &tileLoc,
                                   const AMDAIEDeviceModel &deviceModel) {
   FailureOr<XAie_DmaDesc> dmaTileBd = initDMADesc(deviceModel, tileLoc);
   if (failed(dmaTileBd)) return failure();
-  assert(!block.getOps<UseLockOp>().empty() && "BD block has no lock-usage");
```

Comment:
why this line is deleted, and are we still expecting at least one acquire or one release?

Reply:
Added it: https://github.com/nod-ai/iree-amd-aie/pull/971/files#diff-14d46268eadd9d7fca77f66d72a2ab285546a8f3369b4f3b1ab8c3f5c15185c7R55